app/assets/javascripts/sports.coffee

```diff
@@ -0,0 +1,5 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
+jQuey ->
```

Comment:
This was the "date picker" for the filter sidebar, but Jennifer and I will try another approach